src/librustc_codegen_ironox/base.rs

```diff
@@ -56,11 +57,10 @@ fn codegen_ironox_module<'ll, 'tcx>(
         }
 
         maybe_create_entry_wrapper::<Builder>(&cx);
-
```

Comment:
What would be happening on the `FIXME` line eventually?

Reply:
Probably something like:

```rust
maybe_create_entry_wrapper::<Builder>(&cx);
```

I'll update the comment.

Reply:
Actually, there is nothing else to add... I'll remove the comment.